views/default/tab_client_generate_cert_step3.pdt

```diff
@@ -9,6 +9,24 @@
             </p>
         <?php
         }
+		else if($auth_file_name == 'fileauth.txt'){

```

Comment:
The else statements should begin on the same line as the preceding closing brace
} elseif (...) {
.
.
.
} else {